mono/utils/atomic.c

```diff
@@ -582,5 +582,9 @@ InterlockedCompareExchange64(volatile gint64 *dest, gint64 exch, gint64 comp)
 }
 
 #endif
+#endif
 
+#if defined(HOST_WIN32) && defined(_MSC_VER)
```

Comment:
I had the same idea. Since it could potentially be in source files in any library maybe it could go into glib.h?